draft-zf-community-dev-fund-2-proposal.rst

```diff
@@ -0,0 +1,486 @@
+::
+
+  ZIP: XXXX
+  Title: Dev Fund for ECC, ZF and Zcash Community Grants
+  Owners: Jack Gavigan <jack@zfnd.org>
+  Credits: Eran Tromer
+           Andrew Miller
+           Zooko Wilcox
+           Matt Luongo
+           @aristarchus
+           @dontbeevil
+           Daira Emma Hopwood
+           George Tankersley
+           Josh Cincinnati
+  Status: Draft
+  Category: Consensus Process
+  Created: 2024-07-01
+  License: MIT
+  Discussions-To: 
+  Pull-Request: 
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHALL", "SHALL NOT", "SHOULD", and "MAY" 
+in this document are to be interpreted as described in BCP 14 [#BCP14]_ when,
+and only when, they appear in all capitals.
+
+The term "network upgrade" in this document is to be interpreted as described 
+in ZIP 200 [#zip-0200]_ and the Zcash Trademark Donation and License Agreement
+([#trademark]_ or successor agreement).
+
+The terms "block subsidy" and "halving" in this document are to be interpreted 
+as described in sections 3.10 and 7.8 of the Zcash Protocol Specification.
+[#protocol]_
+
+"Electric Coin Company", also called "ECC", refers to the US-incorporated 
+Zerocoin Electric Coin Company, LLC.
+
+"Bootstrap Project", also called "BP", refers to the US-incorporated 501(c)(3) 
+nonprofit corporation of that name.
+
+"Zcash Foundation", also called "ZF", refers to the US-incorporated 501(c)(3) 
+nonprofit corporation of that name.
+
+"Financial Privacy Foundation", also called "FPF", refers to the Cayman 
+Islands-incorporated foundation company limited by guarantee of that name.
+
+"Section 501(c)(3)" refers to that section of the U.S. Internal Revenue Code 
+(Title 26 of the U.S. Code). [#section501c3]_
+
+"Zcash Community Advisory Panel" refers to the panel of community members 
+assembled by the Zcash Foundation and described at [#zcap]_.
+
+The terms "Testnet" and "Mainnet" are to be interpreted as described in 
+section 3.12 of the Zcash Protocol Specification [#protocol-networks]_.
+
+
+Abstract
+========
+
+This proposal describes a structure for a new Zcash Development Fund (“Dev 
+Fund 2.0”), to be enacted in Network Upgrade 6 and last for 2 years. This 
+Dev Fund would consist of 20% of the block subsidies. 
+
+During the first year, the Dev Fund will be split into 3 slices:
+
+* 28% for the Bootstrap Project (the parent of the Electric Coin Company);
+* 32% for Zcash Foundation;
+* 40% for "Zcash Community Grants", intended to fund large-scale long-term 
+  Projects (administered by the Financial Privacy Foundation, with extra
+  community input and scrutiny).
+
+During the second year, the Dev Fund will be split into 2 slices: 
```

Comment:
This section is subject to change if Bootstrap/ECC declare that they are willing to accept funding for the full two years, instead of just one year.